.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+    - package-ecosystem: "cargo"
+      directory: "/"
+      versioning-strategy: increase
+      schedule:
+          interval: "weekly"
+      ignore:
+          - dependency-name: "*"
+            update-types: ["version-update:semver-patch"]
+
+    - package-ecosystem: "npm"
+      directory: "/"
+      versioning-strategy: increase
+      schedule:
+          interval: "weekly"
+      ignore:
+          - dependency-name: "*"
+            update-types: ["version-update:semver-patch"]
+
+    - package-ecosystem: "github-actions"
+      directory: "/"
+      schedule:
+          interval: "weekly"
```

Comment:
please remove this, we only need cargo one

Reply:
Could you please explain why you don't want GitHub action updates? Thanks 😅

Reply:
ideally we should update github-actions, but updating it sometime can be alot of work. so i prefer to keep what works until it keeps working :)

Reply:
Well... GitHub action files are few tenth/hundred of lines.
It should not be a big deal 😂🤯